MAINTAINERS.md

```diff
@@ -80,12 +82,14 @@ project.
 [John Fastabend]: https://github.com/jrfastab
 [Kornilios Kourtis]: https://github.com/kkourt
 [Laurent Bernaille]: https://github.com/lbernail
+[Louis De Los Santos]: https://github.com/ldelossa
 [Maciej Kwiek]: https://github.com/nebril
 [Martynas Pumputis]: https://github.com/brb
 [Michael Rostecki]: https://github.com/mrostecki
```

Comment:
While at it, could you maybe also adjust Michal's GitHub name? /cc @vadorovsky

```suggestion
[Michael Rostecki]: https://github.com/vadorovsky
```

Reply:
Sure!

Reply:
Should I also update name from Michael to Michal?

Reply:
Yeah, looks like the name wasn't properly spelled in `MAINTAINERS` from the beginning.